modules/ROOT/pages/whatsnew.adoc

```diff
@@ -19,47 +19,33 @@ include::partial$_show_page_header_block.adoc[]
 [WARNING]
 --
 
-Do not deploy Eventing/Sync Gateway until all SGW nodes are at version 3.2.
+Do not deploy Eventing/Sync Gateway until all SGW nodes are at version 3.2+.
 For earlier Sync Gateway versions that do not write import XATTRs, Eventing functions may encounter infinite recursions and duplicate mutations if deployed in a mixed mode SGW environment.
 This occurs if you have a mixed installation consisting of Sync Gateway 3.2 and an earlier version, and you deploy a new Eventing or Sync Gateway function during an upgrade.
 
 --
-== Release 3.2.1 (October 2024)
+== Release 4.0.0 (Q1 2025)
 
-=== Additional Rev Cache Configuration Option
+=== Version Vectors
 
-You can now specify a memory limit on your rev cache with the `rev_cache.max_memory_count_mb` configuration option.
-You can use this option alongside existing configurable limits to the number of documents in the rev cache to reduce the risk of Out of Memory (OOM) issues.
+NOTE: TODO
 
-For more information, see xref:configuration-schema-database.adoc#cache-rev_cache-max_memory_count_mb[Rev Cache Max Memory Count Config Option].
+=== Mobile XDCR Coexistence
 
-== Release 3.2.0 (September 2024)
-
-=== Eventing Support with Sync Gateway 3.2.0+ and Couchbase Server 7.6.3+
-
-Sync Gateway 3.2.0 now supports improved interoperability with Eventing from Couchbase Server version 7.6.3+.
-
-// Reinstate when Server 7.6.4 is available.
-// You can now create Eventing functions with read-write bindings with the source bucket associated with a Sync Gateway database.
-You can use Eventing to handle data changes that happen when applications interact and to integrate with other Couchbase services such as Data, Query and Full Text Search.
-
-For more information, see xref:server-compatibility-eventing.adoc[]
-
-=== Audit Logging Support
-
-Couchbase now provides Audit Logging support for Sync Gateway. Audit Logging provides tools for administrators to track operational irregularities and to support regulatory and security compliance standards, such as link:https://www.hhs.gov/hipaa/index.html[HIPAA] and link:https://soc2.co.uk/soc2[SOC-2].
-For more information, see xref:audit-logging.adoc[].
+NOTE: TODO
 
 == See Also
 
-xref:3.1@sync-gateway:ROOT:whatsnew.adoc[What's new in previous version 3.1].
+xref:3.2@sync-gateway:ROOT:whatsnew.adoc[What's new in previous version 3.2].
```

Comment:
Do we need this bit for 4.0 release?
Hmm, not a problem either way, having it means it's there for when you next to a minor release so 🤷

Reply:
I'm not too sure. Main thought is to try and keep the format consistent since 3.0 🤷🏼 can be something cut after PM review perhaps if not needed.